docs/reference_guide.md

```diff
@@ -390,6 +391,15 @@ Default: 4096
 
 This is the maximum number of keys that can be stored in a map. Increasing the value will consume more memory and increase startup times. There are some cases where you will want to: for example, sampling stack traces, recording timestamps for each page, etc.
 
+## 9. Clang Environment Variables
+
+bpftrace bpftrace parses C header files using libclang, the C interface to Clang.
```

Comment:
There's a double "bpftrace" at the start of the line. Looks good otherwise

Reply:
D'oh!